apps/server/src/services/PlaybackService.ts

```diff
@@ -269,17 +269,26 @@ export class PlaybackService {
     }
   }
 
+  /**
+   * Adds time to current event
+   * @param {number} time time to add in seconds
```

Comment:
you wrote time twice

Reply:
ah, it formats like this for me... should I change it
![image](https://github.com/cpvalente/ontime/assets/19875125/74186a5a-590a-4432-91c2-13c9259fe5b9)
![image](https://github.com/cpvalente/ontime/assets/19875125/a814eef5-1f5e-4f45-b293-42d067917bd8)

Reply:
I usually add a - but didnt remember this was the reason 😅 
up to you, its all good